Install QEMU is failure

I've seen Issue #489 and #592 but I can't resolve. (Sorry, I can't write ruby)
![Picture 3](https://github.com/mistydemeo/tigerbrew/assets/85050558/1bc0da41-336b-49a5-b8d8-3b72aa67bdd3)

From the gist:
```
gcc-5 -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/tcg -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/tcg/i386 -I. -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1 -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/include -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/hw/intc -Ihw/intc -I/usr/local/Cellar/pixman/0.32.6/include/pixman-1 -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/dtc/libfdt -m32 -DOS_OBJECT_USE_OBJC=0 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -Wstrict-prototypes -Wredundant-decls -Wall -Wundef -Wwrite-strings -Wmissing-prototypes -fno-strict-aliasing -fno-common  -Wendif-labels -Wmissing-include-dirs -Wempty-body -Wnested-externs -Wformat-security -Wformat-y2k -Winit-self -Wignored-qualifiers -Wold-style-declaration -Wold-style-definition -Wtype-limits -fstack-protector-strong -I/usr/local/Cellar/libtasn1/4.7/include -I/usr/local/Cellar/nettle/2.7.1/include -I/usr/local/Cellar/gnutls/3.3.18/include -I/usr/local/Cellar/libtasn1/4.7/include -I/usr/local/Cellar/nettle/2.7.1/include -I/usr/local/Cellar/gnutls/3.3.18/include    -I.. -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/target-arm -DNEED_CPU_H -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/include -MMD -MP -MT hw/intc/omap_intc.o -MF hw/intc/omap_intc.d -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -D_REENTRANT -I/usr/local/Cellar/pcre/8.39/include -I/usr/local/Cellar/glib/2.50.1/include/glib-2.0 -I/usr/local/Cellar/glib/2.50.1/lib/glib-2.0/include -I/usr/local/opt/gettext/include -g   -c -o hw/intc/omap_intc.o /private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/hw/intc/omap_intc.c
ld: in /private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/dtc/libfdt/libfdt.a, archive has no table of contents
collect2: error: ld returned 1 exit status
make[1]: *** [qemu-system-alpha] Error 1
make: *** [subdir-alpha-softmmu] Error 2
make: *** Waiting for unfinished jobs....
```


```
g++-5 -I/usr/local/Cellar/pixman/0.32.6/include/pixman-1 -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/dtc/libfdt -m32 -DOS_OBJECT_USE_OBJC=0 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -Wstrict-prototypes -Wredundant-decls -Wall -Wundef -Wwrite-strings -Wmissing-prototypes -fno-strict-aliasing -fno-common  -Wendif-labels -Wmissing-include-dirs -Wempty-body -Wnested-externs -Wformat-security -Wformat-y2k -Winit-self -Wignored-qualifiers -Wold-style-declaration -Wold-style-definition -Wtype-limits -fstack-protector-strong -I/usr/local/Cellar/libtasn1/4.7/include -I/usr/local/Cellar/nettle/2.7.1/include -I/usr/local/Cellar/gnutls/3.3.18/include -I/usr/local/Cellar/libtasn1/4.7/include -I/usr/local/Cellar/nettle/2.7.1/include -I/usr/local/Cellar/gnutls/3.3.18/include    -I.. -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/target-arm -DNEED_CPU_H -I/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/include -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -D_REENTRANT -I/usr/local/Cellar/pcre/8.39/include -I/usr/local/Cellar/glib/2.50.1/include/glib-2.0 -I/usr/local/Cellar/glib/2.50.1/lib/glib-2.0/include -I/usr/local/opt/gettext/include -g  -m32 -framework CoreFoundation -framework IOKit -g   -o qemu-system-aarch64 exec.o translate-all.o cpu-exec.o tcg/tcg.o tcg/optimize.o fpu/softfloat.o disas.o gdbstub-xml.o kvm-stub.o arch_init.o cpus.o monitor.o gdbstub.o balloon.o ioport.o numa.o qtest.o bootdevice.o device_tree.o memory.o savevm.o cputlb.o memory_mapping.o dump.o xen-common-stub.o xen-hvm-stub.o hw/block/virtio-blk.o hw/block/dataplane/virtio-blk.o hw/char/exynos4210_uart.o hw/char/omap_uart.o hw/char/digic-uart.o hw/char/virtio-serial-bus.o hw/cpu/arm11mpcore.o hw/cpu/realview_mpcore.o hw/cpu/a9mpcore.o hw/cpu/a15mpcore.o hw/display/omap_dss.o hw/display/omap_lcdc.o hw/display/pxa2xx_lcd.o hw/display/vga.o hw/dma/omap_dma.o hw/dma/soc_dma.o hw/dma/pxa2xx_dma.o hw/gpio/omap_gpio.o hw/i2c/omap_i2c.o hw/input/pxa2xx_keypad.o hw/input/tsc210x.o hw/intc/armv7m_nvic.o hw/intc/exynos4210_gic.o hw/intc/exynos4210_combiner.o hw/intc/omap_intc.o hw/intc/allwinner-a10-pic.o hw/misc/arm_sysctl.o hw/misc/cbus.o hw/misc/exynos4210_pmu.o hw/misc/imx_ccm.o hw/misc/mst_fpga.o hw/misc/omap_clk.o hw/misc/omap_gpmc.o hw/misc/omap_l4.o hw/misc/omap_sdrc.o hw/misc/omap_tap.o hw/misc/zynq_slcr.o hw/net/virtio-net.o hw/net/vhost_net.o hw/pcmcia/pxa2xx.o hw/scsi/virtio-scsi.o hw/scsi/virtio-scsi-dataplane.o hw/sd/omap_mmc.o hw/sd/pxa2xx_mmci.o hw/ssi/omap_spi.o hw/timer/exynos4210_mct.o hw/timer/exynos4210_pwm.o hw/timer/exynos4210_rtc.o hw/timer/omap_gptimer.o hw/timer/omap_synctimer.o hw/timer/pxa2xx_timer.o hw/timer/tusb6010.o hw/timer/digic-timer.o hw/timer/allwinner-a10-pit.o hw/virtio/virtio.o hw/virtio/virtio-balloon.o hw/arm/boot.o hw/arm/collie.o hw/arm/exynos4_boards.o hw/arm/gumstix.o hw/arm/highbank.o hw/arm/digic_boards.o hw/arm/integratorcp.o hw/arm/kzm.o hw/arm/mainstone.o hw/arm/musicpal.o hw/arm/nseries.o hw/arm/omap_sx1.o hw/arm/palm.o hw/arm/realview.o hw/arm/spitz.o hw/arm/stellaris.o hw/arm/tosa.o hw/arm/versatilepb.o hw/arm/vexpress.o hw/arm/virt.o hw/arm/xilinx_zynq.o hw/arm/z2.o hw/arm/armv7m.o hw/arm/exynos4210.o hw/arm/pxa2xx.o hw/arm/pxa2xx_gpio.o hw/arm/pxa2xx_pic.o hw/arm/digic.o hw/arm/omap1.o hw/arm/omap2.o hw/arm/strongarm.o hw/arm/allwinner-a10.o hw/arm/cubieboard.o target-arm/arm-semi.o target-arm/machine.o target-arm/kvm-stub.o target-arm/translate.o target-arm/op_helper.o target-arm/helper.o target-arm/cpu.o target-arm/neon_helper.o target-arm/iwmmxt_helper.o target-arm/gdbstub.o target-arm/psci.o target-arm/cpu64.o target-arm/translate-a64.o target-arm/helper-a64.o target-arm/gdbstub64.o target-arm/crypto_helper.o ../blockdev.o ../blockdev-nbd.o ../iothread.o ../qdev-monitor.o ../device-hotplug.o ../os-posix.o ../migration.o ../migration-tcp.o ../vmstate.o ../qemu-file.o ../qemu-file-unix.o ../qemu-file-stdio.o ../qemu-char.o ../block-migration.o ../page_cache.o ../xbzrle.o ../migration-exec.o ../migration-unix.o ../migration-fd.o ../accel.o ../bt-host.o ../bt-vhci.o ../dma-helpers.o ../vl.o ../tpm.o ../qmp-marshal.o ../qmp.o ../hmp.o ../qemu-log.o ../tcg-runtime.o ../audio/audio.o ../audio/noaudio.o ../audio/wavaudio.o ../audio/mixeng.o ../audio/coreaudio.o ../audio/wavcapture.o ../backends/rng.o ../backends/rng-egd.o ../backends/rng-random.o ../backends/msmouse.o ../backends/testdev.o ../backends/tpm.o ../backends/hostmem.o ../backends/hostmem-ram.o ../block/stream.o ../block/commit.o ../block/backup.o ../disas/arm.o ../disas/arm-a64.o ../disas/i386.o ../disas/libvixl/utils.o ../disas/libvixl/a64/instructions-a64.o ../disas/libvixl/a64/decoder-a64.o ../disas/libvixl/a64/disasm-a64.o ../hw/audio/es1370.o ../hw/audio/ac97.o ../hw/audio/intel-hda.o ../hw/audio/hda-codec.o ../hw/audio/wm8750.o ../hw/audio/pl041.o ../hw/audio/lm4549.o ../hw/audio/marvell_88w8618.o ../hw/block/block.o ../hw/block/cdrom.o ../hw/block/hd-geometry.o ../hw/block/m25p80.o ../hw/block/nand.o ../hw/block/pflash_cfi01.o ../hw/block/pflash_cfi02.o ../hw/block/ecc.o ../hw/block/onenand.o ../hw/block/nvme.o ../hw/bt/core.o ../hw/bt/l2cap.o ../hw/bt/sdp.o ../hw/bt/hci.o ../hw/bt/hid.o ../hw/bt/hci-csr.o ../hw/char/ipoctal232.o ../hw/char/pl011.o ../hw/char/serial.o ../hw/char/serial-isa.o ../hw/char/serial-pci.o ../hw/char/virtio-console.o ../hw/char/cadence_uart.o ../hw/char/imx_serial.o ../hw/core/qdev.o ../hw/core/qdev-properties.o ../hw/core/fw-path-provider.o ../hw/core/irq.o ../hw/core/hotplug.o ../hw/core/nmi.o ../hw/core/ptimer.o ../hw/core/sysbus.o ../hw/core/machine.o ../hw/core/null-machine.o ../hw/core/loader.o ../hw/core/qdev-properties-system.o ../hw/core/platform-bus.o ../hw/display/ads7846.o ../hw/display/pl110.o ../hw/display/ssd0303.o ../hw/display/ssd0323.o ../hw/display/blizzard.o ../hw/display/exynos4210_fimd.o ../hw/display/framebuffer.o ../hw/display/tc6393xb.o ../hw/dma/pl080.o ../hw/dma/pl330.o ../hw/gpio/max7310.o ../hw/gpio/pl061.o ../hw/gpio/zaurus.o ../hw/i2c/core.o ../hw/i2c/smbus.o ../hw/i2c/smbus_eeprom.o ../hw/i2c/versatile_i2c.o ../hw/i2c/bitbang_i2c.o ../hw/i2c/exynos4210_i2c.o ../hw/ide/core.o ../hw/ide/atapi.o ../hw/ide/qdev.o ../hw/ide/pci.o ../hw/ide/microdrive.o ../hw/ide/ahci.o ../hw/ide/ich.o ../hw/input/hid.o ../hw/input/lm832x.o ../hw/input/pl050.o ../hw/input/ps2.o ../hw/input/stellaris_input.o ../hw/input/tsc2005.o ../hw/intc/pl190.o ../hw/intc/imx_avic.o ../hw/intc/realview_gic.o ../hw/intc/arm_gic_common.o ../hw/intc/arm_gic.o ../hw/ipack/ipack.o ../hw/ipack/tpci200.o ../hw/isa/isa-bus.o ../hw/misc/max111x.o ../hw/misc/tmp105.o ../hw/misc/pci-testdev.o ../hw/misc/arm_l2x0.o ../hw/misc/arm_integrator_debug.o ../hw/misc/a9scu.o ../hw/misc/arm11scu.o ../hw/net/ne2000.o ../hw/net/eepro100.o ../hw/net/pcnet-pci.o ../hw/net/pcnet.o ../hw/net/e1000.o ../hw/net/rtl8139.o ../hw/net/vmxnet_tx_pkt.o ../hw/net/vmxnet_rx_pkt.o ../hw/net/vmxnet3.o ../hw/net/smc91c111.o ../hw/net/lan9118.o ../hw/net/xgmac.o ../hw/net/allwinner_emac.o ../hw/net/cadence_gem.o ../hw/net/stellaris_enet.o ../hw/nvram/eeprom93xx.o ../hw/nvram/fw_cfg.o ../hw/pci-bridge/pci_bridge_dev.o ../hw/pci-bridge/ioh3420.o ../hw/pci-bridge/xio3130_upstream.o ../hw/pci-bridge/xio3130_downstream.o ../hw/pci-bridge/i82801b11.o ../hw/pci-host/pam.o ../hw/pci-host/versatile.o ../hw/pci/pci.o ../hw/pci/pci_bridge.o ../hw/pci/msix.o ../hw/pci/msi.o ../hw/pci/shpc.o ../hw/pci/slotid_cap.o ../hw/pci/pci_host.o ../hw/pci/pcie_host.o ../hw/pci/pcie.o ../hw/pci/pcie_aer.o ../hw/pci/pcie_port.o ../hw/pcmcia/pcmcia.o ../hw/scsi/scsi-disk.o ../hw/scsi/scsi-generic.o ../hw/scsi/scsi-bus.o ../hw/scsi/lsi53c895a.o ../hw/scsi/megasas.o ../hw/scsi/vmw_pvscsi.o ../hw/scsi/esp.o ../hw/scsi/esp-pci.o ../hw/sd/pl181.o ../hw/sd/ssi-sd.o ../hw/sd/sd.o ../hw/sd/sdhci.o ../hw/ssi/pl022.o ../hw/ssi/ssi.o ../hw/ssi/xilinx_spips.o ../hw/timer/arm_timer.o ../hw/timer/arm_mptimer.o ../hw/timer/a9gtimer.o ../hw/timer/cadence_ttc.o ../hw/timer/ds1338.o ../hw/timer/pl031.o ../hw/timer/twl92230.o ../hw/timer/imx_epit.o ../hw/timer/imx_gpt.o ../hw/usb/core.o ../hw/usb/combined-packet.o ../hw/usb/bus.o ../hw/usb/desc.o ../hw/usb/desc-msos.o ../hw/usb/libhw.o ../hw/usb/hcd-uhci.o ../hw/usb/hcd-ohci.o ../hw/usb/hcd-ehci.o ../hw/usb/hcd-ehci-pci.o ../hw/usb/hcd-ehci-sysbus.o ../hw/usb/hcd-xhci.o ../hw/usb/hcd-musb.o ../hw/usb/dev-hub.o ../hw/usb/dev-hid.o ../hw/usb/dev-wacom.o ../hw/usb/dev-storage.o ../hw/usb/dev-uas.o ../hw/usb/dev-audio.o ../hw/usb/dev-serial.o ../hw/usb/dev-network.o ../hw/usb/dev-bluetooth.o ../hw/usb/dev-smartcard-reader.o ../hw/usb/ccid-card-passthru.o ../hw/usb/dev-mtp.o ../hw/usb/host-stub.o ../hw/virtio/virtio-rng.o ../hw/virtio/virtio-pci.o ../hw/virtio/virtio-bus.o ../hw/virtio/virtio-mmio.o ../hw/virtio/dataplane/vring.o ../hw/watchdog/watchdog.o ../hw/watchdog/wdt_i6300esb.o ../net/net.o ../net/queue.o ../net/checksum.o ../net/util.o ../net/hub.o ../net/socket.o ../net/dump.o ../net/eth.o ../net/tap.o ../net/vhost-user.o ../net/tap-bsd.o ../net/slirp.o ../qom/object.o ../qom/container.o ../qom/qom-qobject.o ../qom/cpu.o ../qom/object_interfaces.o ../slirp/cksum.o ../slirp/if.o ../slirp/ip_icmp.o ../slirp/ip_input.o ../slirp/ip_output.o ../slirp/dnssearch.o ../slirp/slirp.o ../slirp/mbuf.o ../slirp/misc.o ../slirp/sbuf.o ../slirp/socket.o ../slirp/tcp_input.o ../slirp/tcp_output.o ../slirp/tcp_subr.o ../slirp/tcp_timer.o ../slirp/udp.o ../slirp/bootp.o ../slirp/tftp.o ../slirp/arp_table.o ../ui/keymaps.o ../ui/console.o ../ui/cursor.o ../ui/qemu-pixman.o ../ui/input.o ../ui/input-keymap.o ../ui/input-legacy.o ../ui/cocoa.o ../ui/curses.o ../ui/vnc.o ../ui/d3des.o ../ui/vnc-enc-zlib.o ../ui/vnc-enc-hextile.o ../ui/vnc-enc-tight.o ../ui/vnc-palette.o ../ui/vnc-enc-zrle.o ../ui/vnc-tls.o ../ui/vnc-auth-vencrypt.o ../ui/vnc-auth-sasl.o ../ui/vnc-ws.o ../ui/vnc-jobs.o trace/generated-helpers.o ../async.o ../thread-pool.o ../nbd.o ../block.o ../blockjob.o ../main-loop.o ../iohandler.o ../qemu-timer.o ../aio-posix.o ../qemu-io-cmds.o ../qemu-coroutine.o ../qemu-coroutine-lock.o ../qemu-coroutine-io.o ../qemu-coroutine-sleep.o ../coroutine-sigaltstack.o ../block/raw_bsd.o ../block/qcow.o ../block/vdi.o ../block/vmdk.o ../block/cloop.o ../block/dmg.o ../block/bochs.o ../block/vpc.o ../block/vvfat.o ../block/qcow2.o ../block/qcow2-refcount.o ../block/qcow2-cluster.o ../block/qcow2-snapshot.o ../block/qcow2-cache.o ../block/qed.o ../block/qed-gencb.o ../block/qed-l2-cache.o ../block/qed-table.o ../block/qed-cluster.o ../block/qed-check.o ../block/vhdx.o ../block/vhdx-endian.o ../block/vhdx-log.o ../block/quorum.o ../block/parallels.o ../block/blkdebug.o ../block/blkverify.o ../block/block-backend.o ../block/snapshot.o ../block/qapi.o ../block/raw-posix.o ../block/null.o ../block/mirror.o ../block/nbd.o ../block/nbd-client.o ../block/sheepdog.o ../block/accounting.o  ../libqemuutil.a ../libqemustub.a   -lz -L/usr/local/Cellar/glib/2.50.1/lib -L/usr/local/opt/gettext/lib -lgthread-2.0 -lglib-2.0 -lintl  -lz -L/usr/local/Cellar/pixman/0.32.6/lib -lpixman-1 -lncurses -framework CoreAudio -ljpeg -lsasl2 -L/usr/local/Cellar/gnutls/3.3.18/lib -lgnutls -L/usr/local/Cellar/gnutls/3.3.18/lib -lgnutls -F/System/Library/Frameworks -framework Cocoa -framework IOKit  -L/private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/dtc/libfdt -lfdt
ld: in /private/tmp/qemu20230814-79525-1xwbzaz/qemu-2.2.1/dtc/libfdt/libfdt.a, archive has no table of contents
collect2: error: ld returned 1 exit status
make[1]: *** [qemu-system-aarch64] Error 1
make: *** [subdir-aarch64-softmmu] Error 2
```

What does your `brew list` output say?

--- a/Library/Formula/qemu.rb
+++ b/Library/Formula/qemu.rb
@@ -31,6 +31,9 @@ class Qemu < Formula
 
   def install
     ENV["LIBTOOL"] = "glibtool"
+    # Need to tell ar(1) to generate a table of contents otherwise ld(1) errors
+    # ld: in dtc/libfdt/libfdt.a, archive has no table of contents
+    ENV["ARFLAGS"] = "srv" if MacOS.version == :leopard
 
     if MacOS.version < :leopard
       # Needed for certain stdint macros on 10.4

